### ISSUE TITLE
Excluding a method changes the binding of an alias

### ISSUE THREAD

**Issue by apblack** [gracelang/minigrace#278]
I think that the spec now does a good job of explaining that [alias and exclude clauses are independent](http://web.cecs.pdx.edu/~grace/doc/lang-spec/#reuse).  But _minigrace_ still gets this wrong.

There is known-failing test at js/tests/known-failing/traitExclusion_test.grace:

```
dialect "minitest"

trait a {
    method border { }
    method size { 3 }
}

trait b {
    method color { }
    method size { 2 }
}

class shape {
    use a 
        alias asize = size
    use b alias bsize = size exclude size
    method draw { }
    method publicBsize { bsize }
    method size { asize + bsize }
}
```

The method `size` should return 2; in the above code it returns 3, because somehow the `exclude size` clause causes `bsize` to be bound to the `size` method from `trait a`.

**Comment by kjx** [comment]
checks... Kernan & slug ("inheritator2") bot seems to get this "right"

**Comment by apblack** [comment]
> Kernan & slug ("inheritator2") both seem to get this "right"

Not surprising really — there is no reason for them to replicate my bug.

This is fixed in commit ec28f11

**Comment by apblack** [comment]
The above test code is now part of t031; the original test is .../tests/retired/t216_traitExclusion_test.grace
